app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -85,11 +104,24 @@ public void updateFromItem(final InfoItem infoItem,
         }
         final CommentsInfoItem item = (CommentsInfoItem) infoItem;
 
+        preferences = PreferenceManager.getDefaultSharedPreferences(itemBuilder.getContext());
+
```

Comment:
@TobiGr it is directly in mid of the changes

Reply:
ah didn't get the correct lines, but just check the first changed file.

Reply:
The lines just below this one.

Reply:
I went through the code and did not see it :D
Looks like I need a second pair of glasses...